apps/default/grafana/deployment.yaml

```diff
@@ -60,6 +60,10 @@ spec:
             - name: providers
               mountPath: /etc/grafana/provisioning/dashboards
 
+            # Root Dashboards
```

Comment:
We should probably (under a separate task) clean up the dashboards that shouldn't be used.

Reply:
I can start on this today